dvc/repo/experiments/queue/celery.py

```diff
@@ -368,12 +371,12 @@ def kill(self, revs: Collection[str], force: bool = False) -> None:
             else:
                 to_kill[queue_entry] = rev
 
-        if missing_revs:
-            raise UnresolvedQueueExpNamesError(missing_revs)
-
         if to_kill:
             self._kill_entries(to_kill, force)
 
+        if missing_revs:
+            raise UnresolvedRunningExpNamesError(missing_revs)
```

Comment:
@karajan1001 I moved this so that valid tasks get removed even if invalid ones were also passed.